packages/terra-button-group/src/ButtonGroup.module.scss

```diff
@@ -58,10 +58,10 @@
 
     &:active.is-selected,
     &:active {
-      background-color: var(--terra-button-group-button-active-background-color, #52585b);
-      border-color: var(--terra-button-group-button-active-border-color, #52585b);
-      box-shadow: var(--terra-button-group-button-active-box-shadow);
-      color: var(--terra-button-group-button-active-color, #fff);
+      background-color: var(--terra-button-group-button-active-is-selected-active-background-color, #52585b);
```

Comment:
I have mixed thoughts on this change, does the linter complain about the original naming of this variable?

Reply:
yeah, this is another case where we either split `active.is-selected` and `active` into separate blocks and have concise themeable variables. Or, keep them in the same block and have long themeable variables

Reply:
I would vote to keep this variable as `--terra-button-group-button-active-background-color`. The is-selected class shouldn't be included with the name because the selector is an `or`. Being active does not mean the button is selected. The button can be `active and is-selected` or just `active`.

The linter might to enforce the double active, but it should be updated to account for `or` selectors. The pseudo class rule is intended for cascading selectors, not sibling `or` selectors. From a dom perspective it's not possible for two elements to be simultaneously active / focuses. Any variables that includes a pseudo class twice may need a little more investigation.

Reply:
Resolved in this commit: https://github.com/cerner/terra-core/pull/2125/commits/f770f2bfa53e9e7633b16aad4a56e621da13302e